portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,7 +89,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentEntity.setProperty("year", Calendar.getInstance().get(Calendar.YEAR));
 
     datastore.put(commentEntity);
-    
+
     response.sendRedirect("/index.html");
   }
+
```

Comment:
nit: We can remove newlines here as well

Reply:
Done.